bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/core/IBigtableDataClient.java

```diff
@@ -62,11 +62,12 @@ Row readModifyWriteRow(ReadModifyWriteRow readModifyWriteRow)
    * Perform an atomic read-modify-write operation on a row.
    *
    * @param readModifyWriteRow a {@link ReadModifyWriteRow} model object.
-   * @return a {@link ApiFuture} of type {@link Row} will be set when request is
+   * @return a {@link ListenableFuture} of type {@link Row} will be set when request is
    *     successful otherwise exception will be thrown.
    * @throws InterruptedException if any.
    */
-  ApiFuture<Row> readModifyWriteRowAsync(ReadModifyWriteRow readModifyWriteRow) throws InterruptedException;
+  //TODO(rahulkql): Once its adapted to v2.models then change the return type to ApiFuture
```

Comment:
here too change as above.

Reply:
I assume you meant for a period to make sentence complete.

Reply:
No, the sentence construction is incorrect. (its and it's have different meanings). I thought, I left the edited sentence too. 
It should read as follows: Once it is adapted to V2.models, change the return type to ApiFuture. (please change wherever this sentence appears)